source/Octopus.Manager.Tentacle.Tests/AssemblyInfo.cs

```diff
@@ -0,0 +1,4 @@
+#if NET8_0_OR_GREATER
+using System.Runtime.Versioning;
+[assembly: SupportedOSPlatform("windows")]
```

Comment:
Things complains about using Windows only APIs for this Windows only feature (Tentacle Manager).